src/ResourceManager/Compute/Commands.Compute/Common/ComputeClientBaseCmdlet.cs

```diff
@@ -40,10 +43,30 @@ public ComputeClient ComputeClient
 
             set { computeClient = value; }
         }
+
         public override void ExecuteCmdlet()
         {
             base.ExecuteCmdlet();
             ComputeAutoMapperProfile.Initialize();
         }
+
+        protected void ExecuteClientAction(Action action)
+        {
+            try
+            {
+                try
```

Comment:
Actually I tried your suggestion before sending the PR, but it doesn't seem to work... I can double check but if that's the case, I would leave it as is...